src/components/views/verification/VerificationShowSas.js

```diff
@@ -69,26 +79,34 @@ export default class VerificationShowSas extends React.Component {
         } else {
             return <div>
                 {_t("Unable to find a supported verification method.")}
-                <DialogButtons
-                    primaryButton={_t('Cancel')}
-                    hasCancel={false}
-                    onPrimaryButtonClick={this.props.onCancel}
-                />
+                <AccessibleButton kind="primary" onClick={this.props.onCancel} className="mx_UserInfo_verify">
+                    {_t('Cancel')}
+                </AccessibleButton>
             </div>;
         }
 
+        let confirm;
+        if (this.state.pending) {
+            const {displayName} = this.props;
+            const text = _t("Waiting for %(displayName)s to verify…", {displayName});
+            confirm = <PendingActionSpinner text={text} />;
+        } else {
+            // FIXME: stop using DialogButtons here once it this component is only used in the right panel verification
+            confirm = <DialogButtons
+                primaryButton={_t("They match")}
+                onPrimaryButtonClick={this.onMatchClick}
+                primaryButtonClassName="mx_UserInfo_verify"
+                cancelButton={_t("They don't match")}
+                onCancel={this.props.onCancel}
+                cancelButtonClass="mx_UserInfo_verify"
+            />;
+        }
+
         return <div className="mx_VerificationShowSas">
             <p>{sasCaption}</p>
-            <p>{_t(
-                "For maximum security, we recommend you do this in person or use another " +
-                "trusted means of communication.",
-            )}</p>
+            <p>{_t("For ultimate security, do this in person or use another way to communicate.")}</p>
```

Comment:
Not finding this text change in the designs, but I assume @nadonomy was consulted...?

It's a bit strange we say "ultimate" here, but still use "maximum" elsewhere...

Reply:
what the heck, I swear the copy used to be different :')
will update to match latest Figma, don't have access to version history to verify my sanity 👎

Reply:
Actually no, I am not crazy: 
![image](https://user-images.githubusercontent.com/2403652/73315549-46aba300-4228-11ea-9b80-9cd9aae65d26.png)
https://www.figma.com/file/wwqBo5oAFfk8XKmtd2YaIs/Cross-signing?node-id=6%3A2550

Reply:
Thanks for investigating. I'd like to have this merged tonight if possible, so it's ready for a practice demo tomorrow morning. Maybe let's land as-is and edit text after once Nad replies?